waspc/data/Generator/templates/react-app/src/operations/index.ts

```diff
@@ -1,12 +1,17 @@
 import api, { handleApiError } from '../api'
 import { HttpMethod } from '../types'
+import { 
+  serialize as superjsonSerialize,
+  deserialize as superjsonDeserialize,
+ } from 'superjson'
 
 export type OperationRoute = { method: HttpMethod, path: string }
 
 export async function callOperation(operationRoute: OperationRoute & { method: HttpMethod.Post }, args: any) {
   try {
-    const response = await api.post(operationRoute.path, args)
-    return response.data
+    const superjsonArgs = superjsonSerialize(args)
+    const response = await api.post(operationRoute.path, superjsonArgs)
```

Comment:
We're still relying on `axios` to fully serialize a "semi-serialized" superjson object under the hood.

More precisely, Axios will turn `{ json: "...", meta: "..." }` (object) into `'{"json": "...", "meta": "..." }'` (string).